content/python/concepts/assert/assert.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'assert'
+Description: 'A keyword for confirming the truthiness of a given statement'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Assert'
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A keyword for confirming the truthiness of a given statement that is often used to facilitate development, document code and debug.
+
+## Syntax
+
+```pseudo
+assert some_condition, assert_message
+```
+
+The `assert` keyword is followed by the statement containing the condition to be tested, such as validating a data type or membership within a given container. An optional message can be added that will be printed when the assertion error is raised.
+
+## Example
+
+The `assert` keyword can be used to validate a value (or evaluate some other condition):
+
+```py
+fav_color = 'blue'
+
+assert fav_color == 'red'
+```
+
+## Codebyte Example
+
+The following example uses `assert` to evaluate if 'mud' is within the following list `things_that_float`:
```

Comment:
```suggestion
The following example uses `assert` to evaluate if 'mud' is within the following `things_that_float` list:
```

Reply:
Not sure I see a benefit in readability by inverting the name and type

Reply:
Alright! You can resolve this convo in that case 👍